### ISSUE TITLE
Inertia for Multi-touch Rotations and Pinch-to-Zoom actions

### ISSUE THREAD

**Issue by EdsterG** [taye/interact.js#225]
I tried searching for the Inertia setting in Multi-touch Rotations but couldn't find it. Shouldn't be difficult to extend the current inertia code to work with Multi-touch Rotations and Pinch-to-zoom actions.

**Comment by taye** [comment]
I felt that some extra physics would be necessary to conserve some angular momentum of the gestures and I wasn't ready to investigate this when I started working on inertia so I just disabled it for gestures. It would be very straight forward to allow inertia simply for each pointer independently and that would probably be good enough.

At the moment all of my focus is on refactoring and modularising the codebase. After that I'd be willing to work on this or accept pull requests. If you have any suggestions or feedback now, I'd be happy to hear them.
